distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.url.mvn.cfg

```diff
@@ -75,7 +79,10 @@ org.ops4j.pax.url.mvn.disableAether=true
 #
 # The following property value will add the system folder as a repo.
 #
-org.ops4j.pax.url.mvn.defaultRepositories=file:${karaf.home}/${karaf.default.repository}@snapshots@id=karaf.${karaf.default.repository}
+org.ops4j.pax.url.mvn.defaultRepositories=\
+    file:${karaf.home}/${karaf.default.repository}@id=system.repository@snapshots, \
+    file:${karaf.data}/kar@id=kar.repository@multi@snapshots, \
```

Comment:
Neither could I.

These settings, however, are the standard ones in Karaf 4.0.8 so they seem the most appropriate to use, even considering that they are undocumented.